src/js/utils/clock.js

```diff
@@ -0,0 +1,40 @@
+define([], function() {
+
+    var AudioClockPolyfill = function() {
+        var currentTime = 0;
+        var time = new Date().getTime();
+
+        var updateClock = function() {
+            var delta = new Date().getTime() - time;
+            if (delta > 250) {
```

Comment:
Yes. Updated the description. AudioContext is only missing in older IE browsers. For those, anytime over a quarter of a second where JavaScript could not be executed will not be added to the running clock. The getting handles precision < 50ms by calculating the delta since start or the last call and adding it to currentTime.